libs/R-TypeLogic/Global/Components/AfkFrequencyComponent.hpp

```diff
@@ -0,0 +1,38 @@
+/*
+** EPITECH PROJECT, 2022
+** R-Type
+** File description:
+** AfkFrequencyComponent
+*/
+
+#ifndef AFKFREQUENCYCOMPONENT_HPP_
+#define AFKFREQUENCYCOMPONENT_HPP_
+
+#include <chrono>
+#include "Component/Component.hpp"
+
+using namespace std::chrono;
+
+namespace ecs
+{
+    /// @brief The AfkFrequency class, how many seconds will be before the entity will be consided has inactive
+    class AfkFrequency : public Component {
+      public:
+        /// @brief The delay before inactivity detection
+        duration<double> frequency;
+
+        /// @brief The base delay, only used to reset the frequency value
+        const duration<double> baseFrequency;
```

Comment:
Shouldn't be private ?

Reply:
All component variable need to be public ;)